spec/support/features/pages/admin_support_portal.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Pages
+  class AdminSupportPortal
+    include Capybara::DSL
+
+    def view_participant_list
```

Comment:
not important but random opinion. i'd probably delete this class and choose to inline. it's not used elsewhere. the method also feels like it conflated trying to both interact with the page and do the page object (which is not quite a page object)